ci/gpu/build.sh

```diff
@@ -59,36 +55,84 @@ conda install -y -c pytorch -c gwerbin \
     "matplotlib" \
     "faker"
 
```

Comment:
Can you add the comment and commands to remove the env pkgs and install a custom pkg version?